Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Steps to Use
+# 1. Download Docker
+# 2. Build with `docker build -t {image tag} .`
+# 3. Run `docker run -t -i -v {local absolute directory}:{container directory} {image tag name} /bin/bash`
+
+FROM ubuntu
+
+RUN apt-get update
+RUN apt-get -y upgrade
```

Comment:
Additionally, the `apt-get install` command should be combined with `apt-get update`. Otherwise, an install can happen despite a failed update.